config/peerpods/podvm/lib.sh

```diff
@@ -644,6 +644,34 @@ function convert_podvm_image_to_vhd() {
 
 }
 
+# Function to check image and convert to raw if needed
```

Comment:
notice there's also convert_qcow2_to_raw now, so I'd either use it or switch the aws code to use this function..